bin/clover/src/cli.ts

```diff
@@ -63,6 +68,33 @@ To generate all specs containing "ECS" or "S3", you can pass some services as ar
         },
       );
     })
+    // load-specs
+    .command(
+      "load-specs [...services:string]",
+      `Load the specs`,
+    )
+    .option("--inferred <file:string>", "Inferred database location", {
+      default: "inferred.json",
+    })
+    .env(
```

Comment:
If we are keeping this, I don't think we need the env vars, right?

Reply:
Oh! I'll remove this.